setup.py

```diff
@@ -55,10 +55,13 @@
 
 version = imp.load_source('pyspectral.version', 'pyspectral/version.py')
 
+description = ('Reading and manipulaing satellite sensor spectral responses and the ' +
+               'solar spectrum, to perfom various corrections to VIS and NIR band data')
```

Comment:
I think you can simplify this by removing the `+` since python concatenates strings split across a line. Not sure that is more clear though.